Formula/lm4tools.rb

```diff
@@ -0,0 +1,23 @@
+class Lm4tools < Formula
+  desc "Tools for TI Stellaris Launchpad boards"
+  homepage "https://github.com/utzig/lm4tools"
+  url "https://github.com/utzig/lm4tools/archive/v0.1.2.tar.gz"
+  sha256 "2619ae4350a3a2190a12f6fae03c86a55975b85f47b891b694f0a7a17f7369a6"
+
+  depends_on "pkg-config" => :build
+  depends_on "libusb"
+
+  def install
+    cd "lm4flash"
```

Comment:
There is a top-level makefile, so we could simply run:

```
    system "make"
    bin.install "lm4flash/lm4flash"
    bin.install "lmicdiusb/lmicdi"
```

Also, could you open an issue with upstream asking that they add an `install` target to their makefile? That way we could do `make install PREFIX=...`

Reply:
There's no top-level makefile in v0.1.2 (the latest release). :( It's a later addition.

Reply:
Upstream issue for the `install` target: https://github.com/utzig/lm4tools/issues/27

Reply:
I added a PR which adds the functionality, I tested on Linux only, would be glad if it looks OK and you can check with Homebrew before I merge. Btw, I could make a release `0.1.3` (or maybe `0.1.4`) afterwards...